src/internal/pretty/pretty.go

```diff
@@ -31,6 +31,9 @@ func Since(timestamp *types.Timestamp) string {
 // Ago pretty-prints the amount of time that has passed since timestamp as a
 // human-readable string, and adds "ago" to the end.
 func Ago(timestamp *types.Timestamp) string {
+	if timestamp == nil {
+		return "N/A"
```

Comment:
just a thought: would "-" be more consistent in our CLI for when we have nothing to show?